tests/src/JIT/opt/Devirtualization/simple.cs

```diff
@@ -0,0 +1,58 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+// Some simple interface call devirtualization cases
+
+interface Ix
+{
+    int F();
+}
+
+interface Iy
+{
+    int G();
+}
+
+interface Iz
+{
+    int H();
+    int I();
+}
+
+public class B : Iy, Ix, Iz
+{
+    public int F() { return 3; }
+    virtual public int G() { return 5; }
+    int Iz.H() { return 7; }
+    int Iz.I() { return 11; }
+}
+
+public class Z : B, Iz
+{
+    new public int F() { return 13; }
+    override public int G() { return 17; }
+    int Iz.H() { return 19; }
```

Comment:
If `Iz` had a couple more methods, `B` and `Z` could implement and re-implement them similar to how they do `F` and `G` (shadowing and overriding).  I don't know/recall what the various C#isms map to in the MSIL for slots and overrides well enough to know if that would be testing anything novel over what you have.